### PR TITLE
index styles

### PR DESCRIPTION
**Pull request by alvinMemphis**
Login page

### CODE REVIEW

File: .idea/shelf/Uncommitted_changes_before_Update_at_27_06_2021_23_19_[Default_Changelist]/shelved.patch [hunk 1]
```diff
@@ -0,0 +1,331 @@
+Index: .idea/.gitignore
```
Comment (Mubangizi) [inline .idea/shelf/Uncommitted_changes_before_Update_at_27_06_2021_23_19_[Default_Changelist]/shelved.patch:1]: Hey @alvinMemphis , just asking what does this file do? and the other XML files that you added?

File: src/index.css [hunk 2]
```diff
@@ -35,3 +30,26 @@ code {
...
   -moz-box-shadow: -1px -2px 18px -4px rgba(0,0,0,0.20);
 }
+body {
+  margin: 0;
+  font-family: 'Nunito Sans', sans-serif;
+  color: var(--primary-color);
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-size: 62.5%;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
```
[the remark this answers is not in the document]
Comment (alvinMemphis) [reply src/index.css]: sorry thought i had

File: .idea/shelf/Uncommitted_changes_before_Update_at_27_06_2021_23_19_[Default_Changelist]/shelved.patch (hunk 1 above)
Comment (alvinMemphis) [reply .idea/shelf/Uncommitted_changes_before_Update_at_27_06_2021_23_19_[Default_Changelist]/shelved.patch:1]: intelij added that stuff ..I'll just git ignore it

**Comment by Mubangizi** [comment]
Okay, I just got access to the login page and here is my feedback.

- The route `/login` is broken, can only be accessed thru the logout button, even when you do a refresh on the login page, it still fails and disappears.
- Then the login word on the login button is invisible. this gets visible on hover, but its alignment is off.
- Make the button clickable, not the login link that's inside the button.
- Reduce the padding on the left and right of the login card but half of what it is now.
- Then you can also give the login card a class of `.BoxShadow` to give it the shadow effect.

Screenshot from 2021-06-29 11-18-08

**Pull request merged**